include/jemalloc/internal/ph.h

```diff
@@ -239,7 +239,7 @@ ph_merge_aux(ph_t *ph, size_t offset, ph_cmp_t cmp) {
 		phn_prev_set(phn, NULL, offset);
 		phn = phn_merge_siblings(phn, offset, cmp);
 		assert(phn_next_get(phn, offset) == NULL);
-		ph->root = phn_merge(ph->root, phn, offset, cmp);
+		phn_merge_ordered(ph->root, phn, offset, cmp);
```

Comment:
I believe the update is wrong, phn_merge_ordered means there's known order for these two node as "assert(cmp(phn0, phn1) <= 0);" shows, however here we just can't make sure the order of ph->root and phn, right?